src/main/java/IonErm/ClassWork/files_13_01/Main.java

```diff
@@ -0,0 +1,50 @@
+package main.java.IonErm.ClassWork.files_13_01;
+
+import java.io.File;
+
+public class Main {
+    public static void main(String[] args) {
+//        File file = new File("resources\\index.txt");
```

Comment:
It is not a good practice to comment such big pieces of code
If code is commented that means it is not used,
Remove it or create a separate method and place it there